src/app.js

```diff
@@ -19,14 +20,16 @@ const { getRemoteTabs, readError } = require(path.resolve(
 const pkg = require(path.resolve(__dirname, '../package.json'));
 const notifier = updateNotifier({ pkg });
 
+const conf = new Configstore(pkg.name, { logAlias: '--color=always' });
+const configLogAlias = conf.get('logAlias');
```

Comment:
I think logAlias should probably be renamed to something more semantic like gitLogArguments or something that makes sense about what it is. It's not technically an alias. 

Also, have you put any thought into how we would validate whether the git log arguments will be valid?

Reply:
Changed the configuration key and const name as you mentioned, it sure does look great now. I already searched for validating pretty formats but haven't found anything. Maybe with a regex ruleset we can validate it and if it fails we can always use default argument instead. Seems like it requires another issue for it

Reply:
What if we just do some type checking to ensure the value we pull from the config is correct? Then we can just wrap it in a try catch when we spawn the git log process.